src/ACEtk/ContinuousEnergyTable/test/ContinuousEnergyTable.test.cpp

```diff
@@ -688,34 +688,34 @@ void verifyChunkU235( const ContinuousEnergyTable& chunk ) {
   CHECK( 0.009 == Approx( delayed.multiplicities().back() ) );
 
   // BDD block
-  CHECK( false == chunk.BDD().empty() );
+  CHECK( false == chunk.BDD()->empty() );
 
-  CHECK( 6 == chunk.BDD().NPCR() );
-  CHECK( 6 == chunk.BDD().numberDelayedPrecursors() );
+  CHECK( 6 == chunk.BDD()->NPCR() );
+  CHECK( 6 == chunk.BDD()->numberDelayedPrecursors() );
 
-  CHECK( 1.33360E-10 == Approx( chunk.BDD().precursorGroupData( 1 ).DEC() ) );
-  CHECK( 2.85300E-08 == Approx( chunk.BDD().precursorGroupData( 6 ).DEC() ) );
+  CHECK( 1.33360E-10 == Approx( chunk.BDD()->precursorGroupData( 1 ).DEC() ) );
+  CHECK( 2.85300E-08 == Approx( chunk.BDD()->precursorGroupData( 6 ).DEC() ) );
 
   // DNED block
-  CHECK( false == chunk.DNED().empty() );
+  CHECK( false == chunk.DNED()->empty() );
 
-  CHECK( 1 == chunk.DNED().LDLW( 1 ) );
-  CHECK( 7944 == chunk.DNED().LDLW( 6 ) );
+  CHECK( 1 == chunk.DNED()->LDLW( 1 ) );
+  CHECK( 7944 == chunk.DNED()->LDLW( 6 ) );
 
-  CHECK( true == std::holds_alternative< OutgoingEnergyDistributionData >( chunk.DNED().energyDistributionData( 1 ) ) );
-  CHECK( true == std::holds_alternative< OutgoingEnergyDistributionData >( chunk.DNED().energyDistributionData( 6 ) ) );
+  CHECK( true == std::holds_alternative< OutgoingEnergyDistributionData >( chunk.DNED()->energyDistributionData( 1 ) ) );
+  CHECK( true == std::holds_alternative< OutgoingEnergyDistributionData >( chunk.DNED()->energyDistributionData( 6 ) ) );
 
-  CHECK( std::nullopt == chunk.DNED().referenceFrame( 1 ) );
-  CHECK( std::nullopt == chunk.DNED().referenceFrame( 6 ) );
+  CHECK( std::nullopt == chunk.DNED()->referenceFrame( 1 ) );
+  CHECK( std::nullopt == chunk.DNED()->referenceFrame( 6 ) );
 
   // PTYPE block
-  CHECK( true == chunk.PTYPE().empty() );
+  CHECK( std::nullopt == chunk.PTYPE() );
```

Comment:
I think you may have done this in the future-past, but this would be a good use of `has_value()`.